app/db/migrations/20200705103525-update-vote.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    queryInterface.addColumn(
```

Comment:
The `addColumn` and `removeColumn` methods are promises, which means that when you are doing two (or more) operations in the same function, the results of both must be handled. Returning the second promise means that one is handled, but the first one is unhandled, and this can cause problems.

An example of how to handle the promises of two or more operations can be done with `Promise.all()`, see the example code in [Sequelize docs: Migration Skeleton](https://sequelize.org/master/manual/migrations.html#migration-skeleton).